src/chainstate/stacks/miner.rs

```diff
@@ -287,6 +310,10 @@ impl<'a> StacksMicroblockBuilder<'a> {
             considered.insert(tx.txid());
         }
         if bytes_so_far + tx_len >= MAX_EPOCH_SIZE.into() {
+            warn!(
+                "Adding microblock tx {} would exceed epoch data size",
```

Comment:
Should this be a warn or debug? It seems like this path is sort of expected behavior: a miner will try to mine until the threshold is met.

Reply:
I'd prefer this not to be `debug` because I think it's useful information to miners -- it's a hint that they may want to slow down their microblock mining so they can get a chance to prioritize higher-fee transactions (instead of burning through their compute budget early while the network is congested).  Would you be okay with `info`?

Reply:
Yeah, info works.

Reply:
I think this still should be changed to `info` -- `warn` indicates that something went wrong, whereas this path is expected behavior if blocks are full.